### ISSUE TITLE
bug: "error processing jobs: Queues not found" when no queues have been created

### ISSUE THREAD

**Issue by TillaTheHun0** [hyper63/hyper-adapter-sqs#18]
I am seeing a lot of `processed jobs: []` in the logs that are drowning out more informative logs.

Perhaps we only log if jobs are processed?

Additionally, I am seeing a lot of `error processing jobs: Queues not found`. This seems like a good log to have, but potentially a red herring, as it sits. If a queue doesn't exist, but there isn't supposed to be any queue created, then it doesn't seem like an error and more of a holding pattern. I think the error would be if the queue was supposed to exist, but it didn't.

In other words, should we check for the existence of the s3 file first? If no file exists, then noop, no error. If the file is present in s3, then we get the queue url and fall into happy path. Thoughts?

**Comment by twilson63** [comment]
1. Gotcha, maybe this should not be logged unless a `LOG_LEVEL` env variable is set to `INFO`
1. Yes, that is a bug, I will dig into it and see why it is happening, did you have any logs created at the time, or was the service completely empty?

**Comment by TillaTheHun0** [comment]
1. Log levels sound good to me.
2. Okay. The service was completely empty, no jobs, no queue. So I would have expected a holding state and not an error in the logs.
